core/src/main/scala/akka/persistence/cassandra/healthcheck/HealthCheckSettings.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.cassandra.healthcheck
+
+import java.time.Duration
+
+import akka.actor.{ ActorSystem, NoSerializationVerificationNeeded }
+import akka.annotation.InternalApi
+import com.typesafe.config.Config
+
+@InternalApi
+private[akka] class HealthCheckSettings(system: ActorSystem, config: Config) extends NoSerializationVerificationNeeded {
+
+  private val healthCheckConfig = config.getConfig("healthcheck")
+
+  val pluginLocation: String = healthCheckConfig.getString("plugin-location")
+
+  val timeout: Duration = healthCheckConfig.getDuration("timeout")
```

Comment:
we typically use `scala.concurrent.duration.FiniteDuration`
```
val timeout: FiniteDuration = healthCheckConfig.getDuration("refresh-interval", TimeUnit.MILLISECONDS).millis
```

Reply:
Done